pkg/registry/apis/folders/register.go

```diff
@@ -156,6 +157,7 @@ func (b *FolderAPIBuilder) UpdateAPIGroupInfo(apiGroupInfo *genericapiserver.API
 	}
 
 	opts.StorageOptions(resourceInfo.GroupResource(), apistore.StorageOptions{
+		EnableFolderSupport:         true,
```

Comment:
This PR has folders enabled for dashboards + folders -- everything else will fail if it tries to write a folder header.

I know some alert configs can also use folders -- but they are not using unified storage yet, so it is not yet relevant